rainmaker_orchestrator/tests/test_kimi_client.py

```
@@ -0,0 +1,457 @@
+                client = KimiClient()
+                
+                call_args = mock_openai.call_args
+                assert call_args[1]['base_url'] == 'http://kimi-linear:8000/v1'
+                assert call_args[1]['api_key'] == 'EMPTY'
+                assert client.model == 'moonshot-v1-8k'
+    
+    def test_init_with_empty_api_key_env(self):
+        """Should fall back to 'EMPTY' when API key env var is empty."""
+        with patch.dict(os.environ, {'KIMI_API_KEY': ''}):
+            with patch('rainmaker_orchestrator.clients.kimi.OpenAI') as mock_openai:
+                client = KimiClient()
```

Ruff (0.14.10)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)